articles/active-directory/saas-apps/ms-confluence-jira-plugin-adminguide.md

```diff
@@ -123,7 +123,29 @@ The following image shows the configuration screen in both Jira and Confluence:
 
    * For App proxy setup , follow the steps on the [Azure AD App Proxy Documentation](../app-proxy/what-is-application-proxy.md).
-
+   * 
+## Release Notes
+
+|Plugin Version   |                                   Release Notes                                           |       Supported JIRA versions       |
+|-----------------|-------------------------------------------------------------------------------------------|-------------------------------------|
+|  1.0.20         |   Bug Fixes:                                                                              | Jira Core and Software:             |
+|                 |   1. JIRA SAML SSO add-on redirects to incorrect URL from mobile browser.                 |  7.0.0 to 9.5.0                     |
+|                 |   2. The mark log section after enabling the JIRA plugin.                                 |                                     |
+|                 |   3. The last login date for a user does not update when user signs in via SSO            |                                     |
+|                 |                                                                                           |                                     |
+|  1.0.19         |   New Feature:                                                                            | Jira Core and Software:             |
+|                 |    Application Proxy Support - Checkbox on the configure plugin screen to toggle the App Proxy mode so as to make the Reply URL editable as per the need to point the App Proxy mode so as to make the Reply URL editable as per the need to point it to the proxy server URL |  6.0 to 9.3.1           |
+|                 |                                                                                           | Jira Service Desk: 3.0.0 to 4.22.1  |
+|                 |                                                                                           |                                     |
+|  1.0.18         |   Bug Fixes:                                                                              | Jira Core and Software:             |
+|                 |    1. Bug fix for the 405 error upon clicking on the Configure button of the Jira Azure AD SSO Plugin.|  6.0 to 9.1.0.          |
+|                 |    2. JIRA server is not rendering the "Project Setting Page" correctly.                  | Jira Service Desk: 3.0.0 to 4.22.1. |
+|                 |    3. JIRA is not forcing Azure AD Login, extra button click was required.                |                                     |
+|                 |    4. We have now resolved the security fix in this version.This will protect you from user impersonation vulnerability.|       |
+|                 |    5. JIRA Service Desk logout issue is resolved.                                         |                                     |
```

Comment:
```suggestion
|                 |    Bug fix for the 405 error upon clicking on the Configure button of the Jira Azure AD SSO Plugin.|  6.0 to 9.1.0.          |
|                 |    JIRA server isn't rendering the "Project Setting Page" correctly.                  | Jira Service Desk: 3.0.0 to 4.22.1. |
|                 |    JIRA isn't forcing Azure AD Login. An extra button click was required.                |                                     |
|                 |    We have now resolved the security fix in this version. This will protect you from user impersonation vulnerability.|       |
|                 |    JIRA Service Desk logout issue is resolved.                                         |                                     |
```